presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -346,6 +346,80 @@ public void testFilterOrder()
                 ImmutableList.of(ImmutableList.of(0, 1, 2, 3)));
     }
 
+    @Test
+    public void testFilterFunctionOrder()
+            throws Exception
+    {
+        // Basic: 1. No input. 2. No input, then with input
+        tester.assertRoundTripWithSettings(
+                INTEGER,
+                IntStream.range(0, 1000).boxed().collect(toList()),
+                ImmutableList.of(
+                        OrcReaderSettings.builder().setFilterFunctions(ImmutableList.of(new TestingNoInputFilterFunction(row -> row % 2 == 0))).build(),
+                        OrcReaderSettings.builder().setFilterFunctions(ImmutableList.of(
+                                new TestingNoInputFilterFunction(row -> row % 2 == 0),
+                                new TestingWithInputIntegerFilterFunction(value -> value % 2 == 0, 0)
+                        )).build()));
```

Comment:
We don't usually put closing brace at the beginning of a line. Put this with the previous line.

Reply:
okay